packages/graphback-cli/src/helpers/init.ts

```diff
@@ -114,4 +114,5 @@ Bootstraping graphql server :dizzy: :sparkles:`)
   await installDependencies(name)
   createDBConfig(name)
   logInfo(postSetupMessage(name))
+  process.exit(0)
```

Comment:
Why this? This seems to be bad pattern for reusing this code

Reply:
After I added the watch command, all other commands got stuck after execution so added this as temporary fix for the workflow